magento/pwa-studio
# Use tokens for color and weight

## Description

With the addition of a `700` font weight to our `@font-face` declaration, any usage of the `bold` keyword is now incorrect. This PR changes those values to the new token for semibold, which is a `600` weight.

Also, this PR sets the color of button text. Currently, it is unset, resulting in bugs such as "disappearing" text in some browsers.

## Related Issue
PWA-650

## Acceptance 

### Verification Stakeholders

- @dpatil-magento 
@davemacaulay 

### Specification

### Verification Steps
1. In Safari, go to `/cart`.
2. Verify that clicking on an accordion header does not change the text color.
3. Verify that no other buttons have regressed.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
